src/MongoDB/Driver/Manager.cpp

```diff
@@ -53,6 +53,7 @@ const StaticString
 	s_MongoDBDriverManager_readpreferencetags("readpreferencetags"),
 	s_MongoDBDriverManager_readPreference("readPreference"),
 	s_MongoDBDriverManager_readPreferenceTags("readPreferenceTags"),
+	s_MongoDBDriverManager_maxStalenessMS("maxStalenessMS"),
```

Comment:
Accepting one canonical form of the option would be consistent with how most PHP libraries (PHPLIB included) operate, but the logic in `mongoc-uri.c` for checking if an option is of a certain type before we invoke a setter is case-insensitive. The case-insensitivity does mean we'd permit some weirdness if the user decided to pass a few variants of the same option with different casing.